package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "pnpm run --parallel --aggregate-output --reporter append-only --filter './packages/**' lint",
     "lint:fix": "pnpm --parallel --aggregate-output --reporter append-only --filter './packages/**' lint:fix",
     "typecheck": "pnpm --parallel --aggregate-output --reporter append-only --filter './packages/**' typecheck",
-    "build": "tsc --build --verbose ./tsconfig.prod.json",
+    "build": "tsc --build --verbose ./tsconfig.prod.json && pnpm run --parallel --filter './packages/**' build",
```

Comment:
I think this is fine because the first stage of build is a "global" monorepo build. Executing build in each repo is just for "local" build (like running ncc/esbuild) so this should be fine to do in parallel.